src/session/session-utils.h

```diff
@@ -50,7 +50,7 @@
 #endif
 
 #define EX 127
-#define DEFAULT_PATH "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+#define DEFAULT_PATH DEFAULT_SESSION_PATH
```

Comment:
Please just drop `DEFAULT_PATH`, and use `DEFAULT_SESSION_PATH` in the two places in session.c.

Reply:
Done.